node/src/components/consensus/highway_core/highway.rs

```diff
@@ -445,6 +484,11 @@ impl<C: Context> Highway<C> {
         match vertex {
             Vertex::Vote(vote) => Ok(self.state.validate_vote(vote)?),
             Vertex::Evidence(_evidence) => Ok(()),
+            Vertex::Endorsements(_endorsements) => {
+                // TODO: Validate against LNC.
+                // We can validate against LNC only after having downloaded all dependencies.
```

Comment:
But it's not here that we'd validate the LNC, is it? That also happens above, in the `Vertex::Vote` case.

Reply:
Oh, right. I meant something else – I wanted to validate whether endorsements don't equivocate.

Reply:
Done.